README.md

```diff
@@ -90,6 +91,7 @@ The tooling has four top-level model sections:
 - `topology`       - The location where servers, clusters, machines, server templates, and other domain-level configuration is specified.
 - `resources`      - The location where resources and services are specified (for example, data sources, JMS, WLDF).
 - `appDeployments` - The location where shared libraries and applications are specified.
+- `kubernetes`     - The location where WLS Kubernetes Operator domain configuration is specified.
```

Comment:
where WLS Kubernetes Operator domain configuration -> where the WLS Kubernetes Operator domain configuration